src/ctl.rs

```diff
@@ -217,10 +219,38 @@ pub mod notify {
 }
 
 /// A 8-bit color with a alpha channel
-#[derive(MDisplay, Constructor)]
+#[derive(Copy, Clone, MDisplay, Constructor)]
 #[display(fmt = "rgba({},{},{},{})", "_0", "_1", "_2", "_3")]
 pub struct Color(u8, u8, u8, u8);
 
+/// A special 8-bit color with an alpha channel for notify
+#[derive(Copy, Clone, MDisplay, Constructor)]
+#[display(fmt = "rgba({:02x}{:02x}{:02x}{:02x})", "_0", "_1", "_2", "_3")]
+pub struct NColorRgba(u8, u8, u8, u8);
+
+/// A special 8-bit color for notify
+#[derive(Copy, Clone, MDisplay, Constructor)]
+#[display(fmt = "rgb({:02x}{:02x}{:02x})", "_0", "_1", "_2")]
+pub struct NColorRgb(u8, u8, u8);
+
+/// An enum for special colors of notify
+#[derive(Copy, Clone)]
+pub enum NColor {
```

Comment:
Instead of creating a new color type could you just make `Color` display with hex by default?